src/commands/moderation/commandTable.ts

```diff
@@ -0,0 +1,117 @@
+import { ApplicationCommandOptionType, ApplicationCommandType, CommandInteraction } from 'discord.js';
+import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
+import { slashCommand, slashCommandStructure, Logger, constantsConfig } from '../../lib';
+
+const data = slashCommandStructure({
+    name: 'generate_command_table',
```

Comment:
Could we make that with `-` instead of `_`, it feels more natural as a command
```suggestion
    name: 'generate-command-table',
```

Reply:
![image](https://github.com/flybywiresim/discord-bot-utils/assets/67422707/8d288991-e099-433e-85ca-845b48499f32)

All other commands currently use `_`. Do we want to refactor all commands?

Reply:
> ![image](https://private-user-images.githubusercontent.com/67422707/299767848-8d288991-e099-433e-85ca-845b48499f32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyMTIyOTgsIm5iZiI6MTcwNjIxMTk5OCwicGF0aCI6Ii82NzQyMjcwNy8yOTk3Njc4NDgtOGQyODg5OTEtZTA5OS00MzNlLTg1Y2EtODQ1YjQ4NDk5ZjMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDE5NDYzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjNTI5YTI4MmUxYTA5ODAxMGUzZGI3YmFkYzc4YjY1NGM0ODk4YWVlOTg5OGNhYmIxYTBiNjczY2MwMDc2ZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.c7rw32VxSbDkSlkUXcDdmj-qrqA06ILQj40Pvq8nHxc)
> 
> All other commands currently use `_`. Do we want to refactor all commands?

Thought we discussed to do so? Overall, `-` is used in natural language and easier to use than `_`, which requires pressing the SHIFT key, seems minor, but it is a difference; and is not something that appears in natural language.

Reply:
Potentially we did - has not happened yet if so!

I can merge this one as is, and I'll go through and refactor all at once in the next PR?

Reply:
sounds good :)